File: python/0/2920.py
numbers = input().split()

for i, number in enumerate(numbers):
    numbers[i] = int(number)


ascending = True
descending = True

for i in range(len(numbers) - 1):
    this_number = numbers[i]
    next_number = numbers[i+1]

    if this_number >= next_number:
        ascending = False
    if this_number <= next_number:
        descending = False

if ascending:
    print('ascending')
elif descending:
    print('descending')
else:
    print('mixed')

File: c/0/2920.c
#include <stdio.h>

int main() {
    int a[8];

    for (int i = 0; i < 8; i++) {
        scanf("%d", &a[i]);
    }

    int ascending = 1;
    int descending = 1;
    for (int j = 0; j < 7; j++) {
        if (a[j] < a[j + 1]) {
            descending = 0;
        } else if (a[j] > a[j + 1]) {
            ascending = 0;
        }
    }

    if (ascending) {
        printf("ascending\n");
    } else if (descending) {
        printf("descending\n");
    } else {
        printf("mixed\n");
    }

    return 0;
}
